src/main/java/uk/gov/pay/adminusers/service/ServiceUpdater.java

```diff
@@ -118,4 +120,9 @@ public Service doUpdateMerchantDetails(String serviceExternalId, UpdateMerchantD
         return ((serviceUpdateRequest, serviceEntity) ->
                 serviceEntity.setCollectBillingAddress(serviceUpdateRequest.valueAsBoolean()));
     }
+
+    private BiConsumer<ServiceUpdateRequest, ServiceEntity> updateCurrentGoLiveStage() {
+        return ((serviceUpdateRequest, serviceEntity) ->
+                serviceEntity.setCurrentGoLiveStage(GoLiveStage.valueOf(serviceUpdateRequest.valueAsString())));
```

Comment:
Can remove one set of brackets here and in the above 2 methods too

Reply:
😉✅